src/Katla/CLI.idr

```diff
@@ -81,7 +81,10 @@ literateCmd = MkCommand
 export
 htmlCmd : Command "html"
 htmlCmd = MkCommand
-  { description = "HTML backend"
+  { description = """
+    HTML backend
+    Usage: html build/ttc/20000101/Katla.idr build/ttc/20000101/Katla.ttm > out.html
```

Comment:
```suggestion
    Usage: html Katla.idr build/ttc/*/Katla.ttm > out.html
```
1. The `.idr` file should not be in a `ttc/` directory
2. No need to be specific about the version number, that leads to fragile scripts